examples/CameraMovementUsage/src/main.cpp

```
@@ -0,0 +1,120 @@
+  public:
+    using std::runtime_error::runtime_error;
+};
+} // namespace Example
+
+int main(void)
+{
+    Engine::Core core;
+
+    core.AddPlugins<Window::Plugin, DefaultPipeline::Plugin, CameraMovement::Plugin>();
+
+    core.RegisterSystem<RenderingPipeline::Init>([](Engine::Core &core) {
```

Cppcheck (2.20.0)

[error] Unhandled exception thrown in function that is an entry point.

(throwInEntryPoint)